src/components/common/Actions.vue

```diff
@@ -24,7 +24,7 @@
     <div class="buttons-right">
       <v-fade-transition hide-on-leave>
         <v-btn id="file-pay-btn" large color="primary"
-          :disabled="!getHaveCorrection || isBusySaving"
+          :disabled="!isFilingChanged || isBusySaving || !isFilingValid"
```

Comment:
Does anything prevent File and Pay when a component is in editing mode?

Reply:
Not yet. This is something that needs to be implemented. Almost all components use 'isEditing' locally within component.

Reply:
It has to be another PR

Reply:
OK. Please create another PR or ticket for this.

Reply:
https://github.com/bcgov/entity/issues/5115